exceptions.rst

```diff
@@ -63,7 +63,15 @@ trapping ALL exceptions:
         raise e
 
 This can be helpful when you have no idea about the exceptions which may
-be thrown by your program.
+be thrown by your program. If you just looking to catch all execptions, 
+but don't actually care about what they are, you can even exclude the 
+``Exception as e`` part. 
+
+Note: catching all exceptions may have unintended consequences because catching 
```

Comment:
"Note:" -> "Note::" 

This will just make sure that it renders in a nice blue box on the website :)